src/components/[guild]/activity/ActivityLogFiltersBar/components/FilterTag/FilterTag.tsx

```diff
@@ -50,10 +52,19 @@ const FilterTag = ({
   const tagFontColor = useColorModeValue("black", undefined)
   const closeBtnFocusBgColor = useColorModeValue("blackAlpha.300", "whiteAlpha.500")
 
-  const { getFilter, updateFilter, removeFilter } = useActivityLogFilters()
+  const { activeFilters, getFilter, updateFilter, removeFilter } =
+    useActivityLogFilters()
+
+  const { activityLogType } = useActivityLog()
+  const isSuperadminActivityLog = activityLogType === "all"
+
+  const filterGuildId = isSuperadminActivityLog
```

Comment:
I think we could use `getFilter("guildId")` here, right? 👀

Reply:
`getFilter` takes the ID of the filter, which is a long string like `cd4bfcf1-ba06-46a2-9515-840476a4024d`, not the query string
should we add a `getFilterByQueryString` function to the hook?

Reply:
Oh, sorry, I completely missed that, then we can leave it as is!